features/step_definitions/batch_action_steps.rb

```diff
@@ -57,6 +57,15 @@
   page.driver.submit form['method'], form['action'], params
 end
 
+When /^I click "(.*?)" and accept confirmation$/ do |link|
+  # Use this implementation instead if poltergeist ever implements it
+  # page.driver.accept_modal(:confirm) { click_link(link) }
+
+  click_link(link)
+  expect(page).to have_content('Are you sure you want to delete these posts?')
+  click_button('OK')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.